### PR TITLE
hotfix(dev-console): Fixes typos in developer console toast messages

### PR DESCRIPTION
**Pull request by deshmukhmayur**
# Fixes ONEPLAT-1374

## Does this PR introduce a breaking change

No.

## Screenshots

N/A

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Did tests pass?

### CODE REVIEW

File: packages/developer-console-spa/src/components/AppSettings.tsx [hunk 1]
```diff
@@ -46,15 +46,14 @@ function AppSettings () {
     updateAppService( editableApp )
       .then( updatedApp => {
         forceRefreshApp( updatedApp );
-        window.OpNotification.success( { subject: 'App Updated Succssfully!' } );
+        window.OpNotification.success( { subject: 'App Updated Successfully!' } );
```
Comment (ghanlohar) [inline packages/developer-console-spa/src/components/AppSettings.tsx:49]:
```suggestion
        window.OpNotification?.success( { subject: 'App Updated Successfully!' } );
```

File: packages/developer-console-spa/src/components/AppSettings.tsx [hunk 2]
```diff
@@ -46,15 +46,14 @@ function AppSettings () {
...
       } )
       .catch( err => {
         window.OpNotification.danger( { subject: 'An error occured when updating the App.', body: 'Please try again later.' } );
```
Comment (ghanlohar) [inline packages/developer-console-spa/src/components/AppSettings.tsx:52]:
```suggestion
        window.OpNotification?.danger( { subject: 'An error occured when updating the App.', body: 'Please try again later.' } );
```

File: packages/developer-console-spa/src/components/AppSettings.tsx [hunk 3]
```diff
@@ -63,7 +62,7 @@ function AppSettings () {
     event.preventDefault();
     deleteAppService( app.id )
       .then( () => {
-        window.OpNotification.success( { subject: 'App Deleted Succssfully!' } );
+        window.OpNotification.success( { subject: 'App Deleted Successfully!' } );
```
Comment (ghanlohar) [inline packages/developer-console-spa/src/components/AppSettings.tsx:65]:
```suggestion
        window.OpNotification?.success( { subject: 'App Deleted Successfully!' } );
```

**Review by ghanlohar — CHANGES_REQUESTED** [review]

File: packages/developer-console-spa/src/components/AppSettings.tsx (hunk 1 above)
Comment (deshmukhmayur) [reply packages/developer-console-spa/src/components/AppSettings.tsx]: Updated.

**Review by riginoommen — APPROVED** [review]

File: packages/developer-console-spa/src/components/AppSettings.tsx [hunk 4]
```diff
@@ -63,11 +62,11 @@ function AppSettings () {
     event.preventDefault();
     deleteAppService( app.id )
       .then( () => {
-        window.OpNotification.success( { subject: 'App Deleted Succssfully!' } );
+        window.OpNotification?.success( { subject: 'App Deleted Successfully!' } );
         history.push( '/' );
       } )
       .catch( err => {
-        window.OpNotification.danger( { subject: 'An error occured when deleting the App.', body: 'Please try again later.' } );
+        window.OpNotification?.danger( { subject: 'An error occured when deleting the App.', body: 'Please try again later.' } );
```
Comment (riginoommen) [inline packages/developer-console-spa/src/components/AppSettings.tsx:69]:
```suggestion
        window.OpNotification?.danger( { subject: 'An error occurred when deleting the App.', body: 'Please try again later.' } );
```

File: packages/developer-console-spa/src/components/AppSettings.tsx [hunk 5]
```diff
@@ -46,15 +46,14 @@ function AppSettings () {
     updateAppService( editableApp )
       .then( updatedApp => {
         forceRefreshApp( updatedApp );
-        window.OpNotification.success( { subject: 'App Updated Succssfully!' } );
+        window.OpNotification?.success( { subject: 'App Updated Successfully!' } );
       } )
       .catch( err => {
-        window.OpNotification.danger( { subject: 'An error occured when updating the App.', body: 'Please try again later.' } );
+        window.OpNotification?.danger( { subject: 'An error occured when updating the App.', body: 'Please try again later.' } );
```
Comment (riginoommen) [inline packages/developer-console-spa/src/components/AppSettings.tsx:52]:
```suggestion
        window.OpNotification?.danger( { subject: 'An error occurred when updating the App.', body: 'Please try again later.' } );
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

**Review by ghanlohar — APPROVED** [review]
LGTM

File: packages/developer-console-spa/src/components/AppSettings.tsx (hunk 4 above)
Comment (deshmukhmayur) [reply packages/developer-console-spa/src/components/AppSettings.tsx]: Oh boy, I need to be more careful😅. My keyboard keeps missing some keystrokes.
Updated the commit.

**Review by riginoommen — APPROVED** [review]

**Pull request merged**